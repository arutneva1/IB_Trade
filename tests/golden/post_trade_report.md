| symbol | side | filled_shares | avg_price | notional | avg_slippage | residual_drift_bps |
| --- | --- | --- | --- | --- | --- | --- |
| AAA | BUY | 100.0000 | 10.00 | 1000.00 | 0.50 | 900.00 |
| BBB | SELL | -50.0000 | 20.00 | -1000.00 | -0.50 | -900.00 |
