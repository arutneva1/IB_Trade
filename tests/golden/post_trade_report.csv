symbol,side,filled_shares,avg_price,notional,avg_slippage,residual_drift_bps
AAA,BUY,100.0,10.0,1000.0,0.5,900.0
BBB,SELL,-50.0,20.0,-1000.0,-0.5,-900.0
